sigle/src/modules/editor/extensions/SlashCommand/CommandList.tsx

```diff
@@ -110,11 +112,25 @@ export const CommandList = forwardRef<CommandListRef, CommandListProps>(
             className={selectedIndex === index ? 'is-selected' : ''}
             onClick={() => selectItem(index)}
           >
-            <Icon width={35} height={35} />
-            <Flex direction="column" css={{ ml: '$2' }}>
-              <Text>{title}</Text>
-              <Text css={{ color: '$gray9', mt: '-8px' }}>{description}</Text>
+            <Flex align="center">
+              <Icon width={35} height={35} />
+              <Flex direction="column" css={{ ml: '$2' }}>
+                <Typography>{title}</Typography>
+                <Typography css={{ color: '$gray9', mt: '-8px' }}>
+                  {description}
+                </Typography>
+              </Flex>
             </Flex>
+            {currentNodeName === title && (
```

Comment:
since updating the active node function to filter through the `slashCommands` which requires a storyId, we would have to pass it through here, so I thought it would be better to instead pass the node name, wdyt?

Reply:
Good idea 👍